core/lib/multivm/src/versions/vm_latest/tests/l1_l2_complete_tx_execution.rs

```diff
@@ -0,0 +1,602 @@
+        0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+        0x00, 0x00,
+    ]);
+    let l2_default_account_bytecode_hash = H256::from([
+        0x10, 0x01, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+        0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+        0x00, 0x00,
+    ]);
+
+    let priority_tx_max_gas_limit = U256::from(72000000);
+
+    let protocol_version = 21;
+
+    let force_deployments_data = vec![];
+    let hex_one_32_bytes = [
+        0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+        0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+        0x00, 0x01,
+    ];
+
+    let action = Action::Add;
+    let mailbox_facet_cut = facet_cut(
+        mailbox_facet_address,
+        &mailbox_facet_contract,
+        action.clone(),
+        true,
+    );
+    let admin_facet_cut = facet_cut(admin_facet_address, &admin_facet_contract, action, false);
+
+    let diamond_cut = compile_initial_cut_hash(
+        vec![admin_facet_cut, mailbox_facet_cut],
+        verifier_params,
+        l2_bootloader_bytecode_hash,
+        l2_default_account_bytecode_hash,
+        deploy_account_address,
+        deploy_account_address,
+        priority_tx_max_gas_limit,
+        diamond_init_address,
+        deploy_account_address,
+    );
+
+    let chain_creation_params = ChainCreationParams {
+        genesis_upgrade: deploy_account_address,
+        genesis_batch_hash: H256::from(hex_one_32_bytes),
+        genesis_index_repeated_storage_changes: U256::from(hex_one_32_bytes),
+        genesis_batch_commitment: H256::from(hex_one_32_bytes),
+        diamond_cut: diamond_cut.clone(),
+        force_deployments_data,
+    };
+
+    let stm_initialize_data = Token::Tuple(vec![
+        Token::Address(deploy_account_address),
+        Token::Address(deploy_account_address),
+        chain_creation_params.to_token(),
+        Token::Uint(U256::from(protocol_version)),
+    ]);
+
+    let init_calldata = stm_contract
+        .function("initialize")
+        .unwrap()
+        .encode_input(&[stm_initialize_data])
+        .unwrap();
+
+    // Deploy Transaprent Upgradeable Proxy
+
+    let (transparent_proxy_contract_code, _transparent_proxy_contract) = read_transparent_proxy();
+    let constructor_data = &[
+        Token::Address(stm_address),
+        Token::Address(deploy_account_address),
+        Token::Bytes(init_calldata),
+    ];
+    let stm_proxy_address = deploy_and_verify_contract(
+        vm,
+        &transparent_proxy_contract_code,
+        Some(constructor_data),
+        &mut deploy_nonce,
+    );
+
+    // Call BH to mint new chain
+
+    let (_bridgehub_contract_bytecode, bridgehub_contract) = read_bridgehub();
+
+    // Initialize BH
+    let initialize_calldata = bridgehub_contract
```

Comment:
you dont really need to call it for the l2 bridgehub as the owner is already set in the constructor

Reply:
I don't deploy the BH, it' been deployed as system contract, so without initialize the test would fail